09-coap/test_spec09.py

```diff
@@ -227,7 +227,7 @@ async def client(host, block_size):
         assert str(response.code) == "2.05 Content"
         assert re.search(
             r"This is RIOT \(Version: .*\) running on a "
-            r"native board with a native MCU\.",
+            r"native64 board with a native MCU\.",
```

Comment:
The board should actually have no impact on the test, so let's remove the specific board and MCU here to make the test not unnecessarily specific (and safe or future migrations ;-))

```suggestion
            r"\S+ board with a \S+ MCU\.",
```